ocs_ci/utility/utils.py

```diff
@@ -2500,7 +2495,7 @@ def skipif_ocs_version(expressions):
         'True' if test needs to be skipped else 'False'
     """
     expr_list = [expressions] if isinstance(expressions, str) else expressions
-    return any(eval(config.ENV_DATA["ocs_version"] + expr) for expr in expr_list)
+    return any(config.ENV_DATA["ocs_version"] + expr for expr in expr_list)
```

Comment:
```suggestion
    return any((config.ENV_DATA["ocs_version"] + expr) for expr in expr_list)
```

Reply:
good catch :-) ... fixed in the new commit